include/libcgroup/config.h

```diff
@@ -29,13 +29,57 @@ extern "C" {
  * and applications can benefit from them.
  */
 
+/*
+ * cgroup systemd settings.
+ */
+struct cgroup_systemd_opts {
+	char slice_name[FILENAME_MAX];
+	char scope_name[FILENAME_MAX];
+	int  setdefault;
+	struct cgroup_systemd_opts *next;
+};
+
 /**
  * Load configuration file and mount and create control groups described there.
  * See cgconfig.conf man page for format of the file.
  * @param pathname Name of the configuration file to load.
  */
 int cgroup_config_load_config(const char *pathname);
 
+/**
+ * Parse the systemd delegation settings from the configuration file
+ * and get the default delegation cgroup name, if set.
+ *
+ * @param pathname Name of the configuration file to load.
```

Comment:
How does this function behave if the default isn't set?

Reply:
@drakenclimber Thank you so much for reviewing the patches. It will return the path without appending the default slice cgroup name.